qiita_db/artifact.py

```diff
@@ -646,6 +649,62 @@ def parents(self):
             return [Artifact(p_id)
                     for p_id in qdb.sql_connection.TRN.execute_fetchflatten()]
 
+    def _create_lineage_graph_from_edge_list(self, edge_list):
+        """Generates an artifact graph from the given `edge_list`
+
+        Parameters
+        ----------
+        edge_list : list of (int, int)
```

Comment:
From the if statement below, edge_list here should be optional and set to None by default in the function definition.